src/amphtml-email/20_Samples/Form_Inside_List.html

```diff
@@ -0,0 +1,73 @@
+<!---
+
+preview: amp4email
+skipValidation: 'true'
+draft: 'true'
+
+--->
+
+<!--
+## Introduction
+
+This sample demonstrates how to update data inside an [`amp-list`](https://www.ampproject.org/docs/reference/components/amp-list)
+based on a user's subsequent input in an [`amp-form`](https://www.ampproject.org/docs/reference/components/amp-form).
+-->
+
+<!-- -->
+<!doctype html>
+<html ⚡4email>
+<head>
+  <meta charset="utf-8">
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <script async custom-element="amp-list" src="https://cdn.ampproject.org/v0/amp-list-0.1.js"></script>
```

Comment:
can you comment for each library what is used for? Similar to [this](https://github.com/ampproject/amp-by-example/blob/master/src/30_Advanced/Autosuggest_form.html)